lib/ansible/plugins/connection/chroot.py

```diff
@@ -88,6 +98,10 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         if not self.chroot_cmd:
             raise AnsibleError("chroot command not found in PATH")
 
+        self.chroot_become_cmd = distutils.spawn.find_executable('sudo')
```

Comment:
use get_bin_path from ansible.module_utils.common.process instead, it can also emit an error when the executable  is missing

Reply:
Ok. I will make the appropriate changes in the near future.

Reply:
Which way is more preferable?

1. Copy chroot plugin code to sudo_chroot and update it there.
2. Inherit chroot.Connection class. Update UID check and split _buffered_exec_command in chroot.py.